vermouth/forcefield.py

```diff
@@ -81,6 +82,7 @@ def read_from(self, directory):
         The provided directory must contain a subdirectory with the same name
         as the force field.
         """
+        self.path = directory
```

Comment:
There may be more than one path used to build a given force field. Also, part of a force field may be added programmatically, which would make the `path` attribute misleading.

Why do you need this attribute? Maybe it would be better to save the path on the blocks themselves?

Reply:
That would be better indeed. I need(ed) it for debugging, making sure I was reading the file I thought I was reading

Reply:
And it's the path to a directory. I thought that should contain everything, even if it's scattered in files within that directory

Reply:
You can run `read_from` multiple times with multiple directories.